passwordinputview/src/main/java/com/matthew/passwordinput/lib/RectF.java

```diff
@@ -0,0 +1,29 @@
+package com.matthew.passwordinput.lib;
+
+import ohos.agp.utils.RectFloat;
+
+/**
+ * RectF class.
+ */
+public class RectF extends RectFloat {
```

Comment:
This is an unused class, please remove it. Even if its required, we can use RectFloat component directly.

Reply:
Done